### DISCUSSION TITLE
Can Stylet Support MEF Bootstrapper

### DISCUSSION THREAD

**Discussion by violet701** [canton7/Stylet#292] [General]
my application is now using caliburn.micro and i want to move to stylet. there are some modules  as plugins make up my app by using MEF. so can stylet support MEF bootstrapper?

**Comment by canton7** [comment]
There are sample bootstrappers for various IoC containers here: https://github.com/canton7/Stylet/tree/master/Bootstrappers . No MEF one, but you should be able to write one easily using one of the others as a template.

**Reply by violet701** [reply]
thank you all the same.
